hack/lib/mesh_resources/authorization-policies/README.md

```diff
@@ -0,0 +1,12 @@
+The resources in this folder are based on https://github.com/openshift-knative/knative-istio-authz-chart.
```

Comment:
The ideal would be that we generate those as part of `make generated-files` so that we pick up changes automatically as we do them on the chart (and we can test them here before releasing them)

Reply:
Makes sense. I'll add this as we will now have a aligned version of the chart 👍